### ISSUE TITLE
Variable name mismatch

### ISSUE THREAD

**Issue by hyeronim** [iNavFlight/blackbox-log-viewer#7]
I think there might be a mismatch in some parameters' name, between INAV and "INAV - Blackbox Explorer". 
For instance:
- in inav source file blackbox.c there is this line

          BLACKBOX_PRINT_HEADER_LINE("min_throttle", "%d", motorConfig()->minthrottle)

- while in the "INAV - Blackbox Explorer" flightlog_parser.js file one finds 

        case "minthrottle":
                that.sysConfig.minthrottle = parseInt(fieldValue, 10);

In other words inav's blackbox writes "min_throttle" in the header, while the viewer expects "minthrottle".

There are other parameters with the same problem.
